utils/queuer.go

```diff
@@ -104,6 +105,22 @@ func (q BlockHeightKVQueue) IsEmpty() bool {
 	return !iter.Valid()
 }
 
+// Keys returns a list with the keys for the values still enqueued
+func (q BlockHeightKVQueue) Keys() []Key {
```

Comment:
Can you add a TODO to convert this to `Iter()`?

Reply:
done